wordpress-mobile/WordPress-Android
# Full screen comments

Fixes #9942 

This PR adds a functionality to expand a comment field to a full-screen view. This is useful when there are long comments that were previously limited by the 4 lines of input height in the Comments detail screen.
 
This PR is mostly based on your changes @theck13. I've added a _section start_ and _section end_ params to the comment dialog fragment (I think it makes sense to keep the cursor position when we expand the comment) and I've fixed the submit button when sending a comment from the reader. It's now correctly enabled when you type in a text and disabled when you remove the text.

To test:
* Go to Reader/Post
* Click on the Comments on a Post
* The Reader comment screen opens
  * Notice that there is an expand button on the left side of the input field
  * Notice that the submit button is disabled
* Write a text in the input field
  * Notice that the submit button is now enabled
* Click on the "Expand" button
  * The comment expands to a full-screen view
  * Notice that the cursor position is kept
* Click on the "Collapse" button
  * The comment collapses back to the inline view
  * Notice that the cursor position is kept
* Click on the "Submit" button
* Comment is posted
  * Notice that the submit button is disabled again

To test 2:
* Go to My Site/Comments
* Click on a Comment
* The Comment Detail screen opens
  * Notice that there is an expand button on the left side of the input field
  * Notice that the submit button is disabled
* Write a text in the input field
  * Notice that the submit button is now enabled
* Click on the "Expand" button
  * The comment expands to a full-screen view
  * Notice that the cursor position is kept
* Click on the "Collapse" button
  * The comment collapses back to the inline view
  * Notice that the cursor position is kept
* Click on the "Submit" button
* Comment is posted
  * Notice that the submit button is disabled again

![untitled](https://user-images.githubusercontent.com/1079756/60581943-239b3c80-9d88-11e9-8539-94f4cafc64a2.gif)

Update release notes:
